### PR TITLE
 Translate the English expressions

### PR DESCRIPTION
**Pull request by gergo85**
And minor code style changes

### CODE REVIEW

File: models/Post.php [hunk 1]
```diff
@@ -61,15 +61,15 @@ class Post extends Model
      * @var array
      */
     public static $allowedSortingOptions = [
```
Comment (LukeTowers) [inline models/Post.php]: Just change this to a static method that uses Lang if you want to translate it. I don't want several component properties each defining their own available options.

Comment (gergo85) [reply models/Post.php]: I got the "Constant expression contains invalid operations" error message. I will search a better solution.

**Comment by gergo85** [comment]
@LukeTowers I updated the code.

File: models/Post.php [hunk 2]
```diff
@@ -61,15 +61,15 @@ class Post extends Model
...
-        'random' => 'Random'
+        'title asc',
+        'title desc',
+        'created_at asc',
+        'created_at desc',
+        'updated_at asc',
+        'updated_at desc',
+        'published_at asc',
+        'published_at desc',
+        'random'
```
Comment (LukeTowers) [inline models/Post.php]: Change this back to key => value, but set the value to the translation key and then just do a simple Lang::get on all the places that use the value.

Comment (gergo85) [reply models/Post.php]: @LukeTowers I made it.

**Comment by LukeTowers** [comment]
Is this good to do a final review on now? @gergo85

**Comment by gergo85** [comment]
@LukeTowers Yes it is.

File: models/Category.php [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 use Str;
 use Model;
-use URL;
+use Url;
```
Comment (LukeTowers) [inline models/Category.php]: Why was this changed?

File: models/Post.php [hunk 4]
```diff
@@ -46,7 +46,7 @@ class Post extends Model
     ];
 
     /**
-     * @var array Attributes to be stored as JSON
+     * @var array Attributes to be stored as JSON.
```
Comment (LukeTowers) [inline models/Post.php]: Please don't add periods to the end of docblocks, and also please stop removing trailing commas from PHP arrays. I prefer to leave the trailing commas in the array so that if the last item is copy-pasted to make a new one without updating commas the array is still valid.

Comment (gergo85) [reply models/Post.php]: Alright.

File: models/Category.php (hunk 3 above)
Comment (gergo85) [reply models/Category.php]: Because of https://github.com/rainlab/blog-plugin/blob/master/models/Post.php#L4

**Comment by gergo85** [comment]
If you have any comments, please write an example.
